pkgs/development/libraries/geos/default.nix

```diff
@@ -10,23 +9,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "geos";
-  version = "3.11.2";
```

Comment:
Move the existing 3.11.2 to new `geos311` derivation.

Reply:
Thank you @willcohen , I am planning to do exactly this. In addition, we need to provide shapely 1.8 built with GEOS 3.11 which can work for kikit.